tailwind.config.js

```diff
@@ -5,7 +5,14 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
```

Comment:
If you check the resource that I shared https://stackoverflow.com/a/64175955

In that they use first get all the defaultcolors and than add the custom colors. 

In you case you are directly adding custom colors and as a result we won't be able to use any color which tailwind provides by default.

Check the link again and try to apply solution similar to that.

Reply:
@rt4914 thank you for this observations.

It was an honest mistake.

I will rectify it. 🤝🏾

Reply:
> If you check the resource that I shared https://stackoverflow.com/a/64175955
> 
> In that they use first get all the defaultcolors and than add the custom colors.
> 
> In you case you are directly adding custom colors and as a result we won't be able to use any color which tailwind provides by default.
> 
> Check the link again and try to apply solution similar to that.

done too